### PR TITLE
feat(client): trace client did

### PR DESCRIPTION
**Pull request by saitofun**
#248

### CODE REVIEW

**Review by CoderZhi — APPROVED** [review]

**Comment by CoderZhi** [comment]
will the database be updated automatically?

File: persistence/postgres.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ import (
 type message struct {
 	gorm.Model
 	MessageID      string `gorm:"index:message_id,not null"`
+	ClientDID      string `gorm:"column:client_did;index:client_did,not null,default:''"`
```
Comment (huangzhiran) [inline persistence/postgres.go]: maybe need change the index name to "message_fetch", for message aggrate query

Comment (huangzhiran) [inline persistence/postgres.go]: column:client_did maybe don't need set, it is the gorm default strategy

Comment (saitofun) [reply persistence/postgres.go]: The column name assigned by gorm `client_d_id` is not expected

**Pull request merged**